packages/codegen/src/generators/react-admin/common/mergeI18nMessagesWithLocaleFile.ts

```diff
@@ -0,0 +1,124 @@
+import deepmerge from "deepmerge";
+import * as fs from "fs/promises";
+import j from "jscodeshift";
+import * as path from "path";
+import { EntityMessages } from "../../../building-blocks/stages/template-model/pieces/getEntityMessages";
+import { parseTs } from "../../../building-blocks/util/parseTs";
+import { ScreenPaths } from "./generatorPaths";
+
+/**
+ * Find locale file by localeName in file system of generated app
+ * then find variable with localeMessages by localeName
+ * then find "resources" property inside variable
+ * then merge default messages with existing messages to locale file in file system
+ * 
+ * @param screenPaths - contain information about i18n messages dir path
+ * @param localeName - using to get locale file in i18n messages dir and find variable with locale messages inside
+ * @param defaultMessages - messages that should by in generated app, them can be overridden later in generated app
+ */
+export async function mergeI18nMessagesWithLocaleFile(
+  screenPaths: ScreenPaths,
+  localeName: string,
+  defaultMessages: EntityMessages
+) {
+  const localePath = path.join(
+    screenPaths.localeMessagesPath,
+    `${localeName}.ts`
+  );
```

Comment:
Not sure that we need so many lines for this code, read better in one line
```suggestion
  const localePath = path.join(screenPaths.localeMessagesPath, `${localeName}.ts`);
```

There are also several similar cases in this file.

Reply:
It's prettier formatter

Reply:
Prettier is not standard for code in 'codegen' package. The only place where it must - is generated code. 
Code readability has higher priority than prettier formatting for code in 'codegen'. If you are using prettier formatting, please review your code after it. 
Also I can suggest increase line length in prettier settings (printWidth: 100). This value we are using in generated code formatting  <https://github.com/Amplicode/amplicode-frontend/blob/master/packages/codegen/src/building-blocks/stages/configuring/defaultConfigureGenerator.ts#L62>